sphinx_rtd_theme/locale/sphinx.pot

```diff
@@ -0,0 +1,162 @@
+msgid "Edit on GitHub"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:50 sphinx_rtd_theme/breadcrumbs.html:52
+msgid "Edit on Bitbucket"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:57 sphinx_rtd_theme/breadcrumbs.html:59
+msgid "Edit on GitLab"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:62 sphinx_rtd_theme/breadcrumbs.html:64
+msgid "View page source"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:74 sphinx_rtd_theme/footer.html:5
+msgid "Next"
+msgstr ""
+
+#: sphinx_rtd_theme/breadcrumbs.html:77 sphinx_rtd_theme/footer.html:8
+msgid "Previous"
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:19
+#, python-format
+msgid "&copy; <a href=\"%(path)s\">Copyright</a> %(copyright)s."
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:21
+#, python-format
+msgid "&copy; Copyright %(copyright)s."
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:26
+#, python-format
+msgid ""
+"\n"
+"        <span class=\"build\">\n"
+"          Build\n"
+"          <a href=\"%(build_url)s\">%(build_id)s</a>.\n"
+"        </span>\n"
+"      "
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:33
+#, python-format
+msgid ""
+"\n"
+"        <span class=\"commit\">\n"
+"          Revision <code>%(commit)s</code>.\n"
+"        </span>\n"
+"      "
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:39
+#, python-format
+msgid "Last updated on %(last_updated)s."
+msgstr ""
+
+#: sphinx_rtd_theme/footer.html:46
+msgid ""
+"Built with <a href=\"http://sphinx-doc.org/\">Sphinx</a> using a <a "
+"href=\"https://github.com/rtfd/sphinx_rtd_theme\">theme</a> provided by "
+"<a href=\"https://readthedocs.org\">Read the Docs</a>"
+msgstr ""
+
+#: sphinx_rtd_theme/layout.html:37
+#, python-format
+msgid "Search within %(docstitle)s"
+msgstr ""
+
+#: sphinx_rtd_theme/layout.html:61
+msgid "About these documents"
+msgstr ""
+
+#: sphinx_rtd_theme/layout.html:64
+msgid "Index"
+msgstr ""
+
+#: sphinx_rtd_theme/layout.html:67 sphinx_rtd_theme/search.html:11
+msgid "Search"
+msgstr ""
+
+#: sphinx_rtd_theme/layout.html:70
+msgid "Copyright"
+msgstr ""
+
+#: sphinx_rtd_theme/layout.html:107
+msgid "Logo"
+msgstr ""
+
+#: sphinx_rtd_theme/search.html:26
+msgid ""
+"Please activate JavaScript to enable the search\n"
+"      functionality."
+msgstr ""
+
+#: sphinx_rtd_theme/search.html:33
+msgid "Search Results"
+msgstr ""
+
+#: sphinx_rtd_theme/search.html:35
+msgid ""
+"Your search did not match any documents. Please make sure that all words "
+"are spelled correctly and that you've selected enough categories."
+msgstr ""
+
+#: sphinx_rtd_theme/searchbox.html:4
+msgid "Search docs"
+msgstr ""
+
+#: sphinx_rtd_theme/versions.html:11
+msgid "Versions"
+msgstr ""
+
+#: sphinx_rtd_theme/versions.html:17
+msgid "Downloads"
+msgstr ""
+
+#: sphinx_rtd_theme/versions.html:23
+msgid "On Read the Docs"
+msgstr ""
+
+#: sphinx_rtd_theme/versions.html:25
+msgid "Project Home"
+msgstr ""
+
+#: sphinx_rtd_theme/versions.html:28
+msgid "Builds"
+msgstr ""
+
+#: sphinx_rtd_theme/versions.html:32
```

Comment:
The URL and Read the Docs should not be part of the translated message.

Reply:
@Blendify, you are right! fixedthis string :+1: thanks for your feedback :nerd_face: